src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -387,11 +418,86 @@ public bool MoreExpanded
             }
         }
 
+        private List<HostComboboxEntry> knownHosts;
+
+        /// <summary>
+        /// Know hosts received from package manager.
+        /// Data binded to the multi-selection host dependency option.
+        /// </summary>
+        public List<HostComboboxEntry> KnownHosts
+        {
+            get { return knownHosts; }
+            set
+            {
+                if (knownHosts != value)
+                {
+                    knownHosts = value;
+                    RaisePropertyChanged("KnownHosts");
```

Comment:
for these raise property changed can you use `nameof` ?

Reply:
Sure let me update